### ISSUE TITLE
tkn version command display pipeline version as devel instead of 0.14.0

### ISSUE THREAD

**Issue by savitaashture** [tektoncd/pipeline#2898]
# Expected Behavior
tkn version should display pipeline version as `0.14.0` when installed from released yaml

# Actual Behavior
tkn version displaying pipeline version as `devel` when installed from released yaml

# Steps to Reproduce the Problem

1. Install pipeline from latest release 
`kubectl apply -f https://github.com/tektoncd/pipeline/releases/download/v0.14.0/release.yaml`
2. tkn version
3. output
```
tkn version
Client version: dev
Pipeline version: devel
```

# Additional Info

- Kubernetes version:

```
Client Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.0", GitCommit:"641856db18352033a0d96dbc99153fa3b27298e5", GitTreeState:"clean", BuildDate:"2019-03-25T15:53:57Z", GoVersion:"go1.12.1", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.2", GitCommit:"c97fe5036ef3df2967d086711e6c0c405941e14b", GitTreeState:"clean", BuildDate:"2019-10-15T19:09:08Z", GoVersion:"go1.12.10", Compiler:"gc", Platform:"linux/amd64"}
```

- Tekton Pipeline version:

```
0.14.0
```

**Comment by ckadner** [comment]
https://github.com/tektoncd/cli/issues/1055#issuecomment-655305701

> I fixed the release.yaml for the time being (and there is a bugfix for this not to happen in the next releases, will be in a 0.14.1 too).

---
> 
> ```Bash
> $ tkn version
> Client version: 0.10.0
> Pipeline version: v0.14.0
> Triggers version: unknown
> ```

**Comment by vdemeester** [comment]
#2900 has been merged, should also be fixed when doing 0.14.1
/close
